.github/workflows/checks.yml

```diff
@@ -43,3 +43,34 @@ jobs:
         with:
           header: Benchmark
           message: ${{ steps.benchmark.outputs.report }}
+          recreate: true
+  type-diff:
+    name: Type-diff with Node ${{ matrix.node }} in ${{ matrix.os }}
```

Comment:
I'm not sure I follow. Do you have an example?

Reply:
```suggestion
    name: Type-diff with Node 18 in ubuntu
```

There's only 1 run happening, so there's not as much value in variables.

Reply:
I see what you mean. Yeah, we could inline the variables instead, but I leaned towards leaving them in case we want to introduce more options in the future. There isn't much difference in the resulting content in both cases.